pom.xml

```diff
@@ -225,13 +231,15 @@
               <encoding>UTF-8</encoding>
               <consoleOutput>true</consoleOutput>
               <failsOnError>true</failsOnError>
-              <linkXRef>false</linkXRef>
             </configuration>
             <goals>
               <goal>check</goal>
             </goals>
           </execution>
         </executions>
+        <configuration>
+          <linkXRef>false</linkXRef>
```

Comment:
Just wondering why we don't want this. I know we have this in our other helios projects, but I don't know why. I see [docs here](https://maven.apache.org/plugins/maven-checkstyle-plugin/checkstyle-mojo.html) say

> Link the violation line numbers to the source xref. Will link automatically if Maven JXR plugin is being used